op-batcher/metrics/metrics.go

```diff
@@ -481,9 +493,9 @@ func (m *Metrics) ClearAllStateMetrics() {
 	m.pendingBlocksBytesCurrent.Set(0)
 }
 
-// estimateBatchSize returns the estimated size of the block in a batch both with compression ('daSize') and without
+// EstimateBatchSize returns the estimated size of the block in a batch both with compression ('daSize') and without
 // ('rawSize').
-func estimateBatchSize(block *types.Block) (daSize, rawSize uint64) {
+func EstimateBatchSize(block *types.Block) (daSize, rawSize uint64) {
 	daSize = uint64(70) // estimated overhead of batch metadata
 	rawSize = uint64(70)
```

Comment:
Note that we're adding this constant overhead for _each_ block. We should double check that this is a valid estimation of the _compressed_ overhead so that we're not hugely overestimating.

Reply:
This could be very inaccurate for span batches, where the metadata overhead is only weakly dependent on the number of blocks https://specs.optimism.io/protocol/delta/span-batches.html#span-batch-format.